### PR TITLE
Create post page, posts page image and bee logo

### PR DESCRIPTION
**Pull request by katladva**
Posts page post image and default image added, small bee logo added to the post. Created create_post.html template, added form for new post. At the moment create url is not working.

### CODE REVIEW

File: templates/base.html
Comment (henriits) [inline templates/base.html:1]: static is added 2x

File: templates/base.html [hunk 1]
```diff
@@ -21,7 +21,9 @@
           integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <!-- Core CSS-->
     <link href="{% static 'css/styles.css' %}" rel="stylesheet"/>
-    <title>SocialVibes</title>
+    <!-- New post -->
+    <link rel="stylesheet" type="text/css" href="{% static 'css/post-style.css' %}"
```
Comment (henriits) [inline templates/base.html]: remove this

File: templates/base.html [hunk 2]
```diff
@@ -21,8 +21,6 @@
           integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <!-- Core CSS-->
     <link href="{% static 'css/styles.css' %}" rel="stylesheet"/>
-    <!-- New post -->
-    <link rel="stylesheet" type="text/css" href="{% static 'css/post-style.css' %}"
     <title>HumbleBee</title>
```
Comment (henriits) [inline templates/base.html:24]: good

**Review by henriits — APPROVED** [review]

File: templates/base.html (hunk 1 above)
Comment (katladva) [reply templates/base.html]: Done

**Pull request merged**